SethY8s/subspace-project
# Added Base Project

The base project is now working properly. 

What was added:
-Smart contract
-Deployment code
-Next frontend
-Type safety precautions.

How to test:

`npm install`

`cd smart-contract && npm install`

`hardhat clean && hardhat compile`

`cd .. && npm run dev`

The project should now be fully functional with types
